src/patternfly/components/Dropdown/dropdown-group-title.hbs

```diff
@@ -0,0 +1,10 @@
+<{{#if dropdown-group-title--type}}{{dropdown-group-title--type}}{{else}}h2{{/if}} class="pf-c-dropdown__group-title{{#if dropdown-group-title--modifier}} {{dropdown-group-title--modifier}}{{/if}}"
```

Comment:
Looks like these were `<h1>` before, and are `<h1>` by default in the react dropdown.